medusa/helpers/__init__.py

```diff
@@ -1790,8 +1790,6 @@ def get_broken_providers():
 
 def is_already_processed_media(full_filename):
     """Check if resource was already processed."""
-    if not is_media_file(str(full_filename)):
```

Comment:
@medariox is this alright?

Reply:
this is my code. I don't remember why I added `str` 
I tested the is_media_file in the transmission code and it worked without it

Reply:
btw this is what transmission returns for each torrent.:

I get the `files` and check if is_media_file

```
{"activityDate":1511748601,"errorString":"","files":[{"bytesCompleted":2953150,"length":2953150,"name":"The.Blacklist.S05E08.1080p.WEB.x264-TBS/Screens/screen0001.png"},{"bytesCompleted":2166534,"length":2166534,"name":"The.Blacklist.S05E08.1080p.WEB.x264-TBS/Screens/screen0002.png"},{"bytesCompleted":1854279,"length":1854279,"name":"The.Blacklist.S05E08.1080p.WEB.x264-TBS/Screens/screen0003.png"},{"bytesCompleted":1626775,"length":1626775,"name":"The.Blacklist.S05E08.1080p.WEB.x264-TBS/Screens/screen0004.png"},{"bytesCompleted":1351198618,"length":1351198618,"name":"The.Blacklist.S05E08.1080p.WEB.x264-TBS/the.blacklist.s05e08.1080p.web.x264-tbs.mkv"},{"bytesCompleted":50,"length":50,"name":"The.Blacklist.S05E08.1080p.WEB.x264-TBS/the.blacklist.s05e08.1080p.web.x264-tbs.nfo"}],"hashString":"1a428b281eb19aba9cea5ca8837598179126ba38","isFinished":false,"isStalled":false,"name":"The.Blacklist.S05E08.1080p.WEB.x264-TBS","percentDone":1,"seedIdleLimit":10080,"seedRatioLimit":3,"status":0,"uploadRatio":0.9960}
```